### ISSUE TITLE
MDWiki  (mdwiki and mdwiki_app) runs both failed this month (Nov 23)

### ISSUE THREAD

**Issue by Jaifroid** [openzim/zim-requests#744]
They both failed on 5th November: see https://farm.openzim.org/recipes/mdwiki and https://farm.openzim.org/recipes/mdwiki_app. Could someone restart them to try again? Custom apps for Windows and Linux are built with these versions.

@Popolechien @kelson42

**Comment by Popolechien** [comment]
Restarted!

**Comment by Jaifroid** [comment]
Thanks -- unfortunately, they failed again...

@WikiDocJames is there an issue on your end? We had a run of 5 good months, but now they seem to be failing again systematically.

**Comment by WikiDocJames** [comment]
Okay will look into it

J

On Sat, Nov 11, 2023 at 12:02 PM Jaifroid ***@***.***> wrote:
>
> Thanks -- unfortunately, they failed again...
>
> @WikiDocJames is there an issue on your end? We had a run of 5 good months, but now they seem to be failing again systematically.
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you were mentioned.Message ID: ***@***.***>



-- 
James Heilman
MD, CCFP-EM, Wikipedian

**Comment by tim-moody** [comment]
Looks like an upgrade at mdwiki broke the api. It has now been fixed, and I ran mwoffliner against it which ran to completion with no complaints.

BUT the video is not working at the top right and also at the bottom of that block

last month https://iiab.me/kiwix/mdwiki_en_all_2023-10/A/Gout
this month https://iiab.me/kiwix/mdwiki_en_all_2023-11/A/Gout

Is there a change to mwoffliner that could explain this? Or can someone suggest something in the api?

For reference this is the docker command:

```
docker run -it \
  -v $(pwd)/output:/output:rw \
  -v $(pwd)/lists:/lists \
  -v $(pwd)/scripts:/scripts \
  --name $1 \
  --cpu-shares 3072 \
  --memory-swappiness 0 \
  --memory 6g \
  ghcr.io/openzim/mwoffliner:latest /bin/bash

```
and this is the mwoffliner command:

```
mwoffliner \
  --adminEmail="info@iiab.me" \
  --articleList="http://offline.mdwiki.org/nonwiki/lists/mdwikimed.tsv" \
  --customMainPage="App/IntroPage" \
  --customZimTitle="MDWiki Medical Encyclopedia" \
  --customZimDescription="Collection of Healthcare Content mdwiki.org." \
  --customZimFavicon="https://en.wikipedia.org/static/images/project-logos/enwiki.png" \
  --filenamePrefix="mdwiki_en_all" \
  --format="novid:maxi" --format="" \
  --mwUrl="http://offline.mdwiki.org/" \
  --osTmpDir="/dev/shm" \
  --outputDirectory="/output" \
  --speed="1" \
  --webp \
  --verbose

```

**Comment by Jaifroid** [comment]
> Is there a change to mwoffliner that could explain this? Or can someone suggest something in the api?

@tim-moody, I'm not sure, but pinging @kelson42, who would know the answer re video and mwoffliner. Many thanks for your investigations.

**Comment by kelson42** [comment]
> > Is there a change to mwoffliner that could explain this? Or can someone suggest something in the api?
> 
> @tim-moody, I'm not sure, but pinging @kelson42, who would know the answer re video and mwoffliner. Many thanks for your investigations.

Zimfarm scraper versions are tagged! No change can happen "just like this"...

**Comment by tim-moody** [comment]
@kelson42 thanks. I guess I should have inferred that the same 1.13.0 version was used last month by looking at github to see when it was released.

**Comment by tim-moody** [comment]
I did a new run and mwoffliner completed without complaints and videos play.

**Comment by kelson42** [comment]
https://farm.openzim.org/pipeline/bdd9cecb-39a1-42e1-92ee-e8ba218eb493

**Comment by Jaifroid** [comment]
> I did a new run and mwoffliner completed without complaints and videos play.

That's great new, thanks! I'll make a new release as soon as I get a chance.

**Comment by tim-moody** [comment]
@kelson42 that's fantastic and thanks for doing another run this month. I did a quick check and the Gout videos play and Yellow Fever, the last error we had, was fine.

I confess I'm a little confused that the farm zim is

6317604289 Nov 27 23:42 mdwiki_en_all_2023-11.zim [EST]

whereas mine was

7142706160 Nov 27 05:06 mdwiki_en_all_2023-11.zim [UTC], 700K larger

Is there something in the syntax I gave above in this issue that could change compression? Of course it's also possible that the mirrored articles from EN WP changed.
